common/men_at_arms_types/00_maa_types.txt

```diff
@@ -423,6 +423,33 @@ massive_aerial_unit = {
 	
 }
 
+riflemen = {
+	type = archers
+	
+	damage = 65
+	toughness = 10
```

Comment:
Why it has less toughness than crossbowmen?
I would say it should have the same.